.github/workflows/e2e.yaml

```
@@ -31,8 +31,11 @@ jobs:
         if: matrix.cable_driver == 'wireguard'
         run: |
           sudo add-apt-repository -y ppa:wireguard/wireguard # add WireGuard support
-          sudo apt-get update
-          sudo apt-get install linux-headers-`uname -r` wireguard -y
+          sudo apt update
+          # temporary solution for Ubuntu 16.04 azure-1020
+          sudo apt-get install linux-headers-$(uname -r) wireguard  -y || true # because wireguard installation it will fail
+          sudo cp queueing.h  /usr/src/wireguard-1.0.20200506/queueing.h
+          sudo dpkg-reconfigure wireguard-dkms
```

```suggestion
          if ! sudo apt-get install linux-headers-$(uname -r) wireguard  -y; then
              sudo curl -L https://git.zx2c4.com/wireguard-linux/plain/drivers/net/wireguard/queueing.h -o /usr/src/wireguard-1.0.20200506/queueing.h
              sudo dpkg-reconfigure wireguard-dkms
          fi
```

Perhaps something like this?